src/script/common/c_content.cpp

```diff
@@ -1990,3 +1991,54 @@ HudElementStat read_hud_change(lua_State *L, HudElement *elem, void **value)
 	}
 	return stat;
 }
+
+/******************************************************************************/
+
+static const char *collision_type_str[] = {
+	[COLLISION_NODE] = "node",
```

Comment:
this may be a non-standard feature, discussed on irc

http://irc.minetest.net/minetest-dev/2020-04-15#i_5669442
https://stackoverflow.com/a/17581108

Reply:
we should see if visual studio support it. For gcc & clang it's fine

Reply:
MSVC does not support this: https://godbolt.org/z/728fMP